hatch_build.py

```diff
@@ -428,6 +428,9 @@
     "cryptography>=41.0.0",
     "deprecated>=1.2.13",
     "dill>=0.2.2",
+    # Required for python 3.8 and 3.9 to work with new annotations styles. Check package
+    # description on PyPI for more details: https://pypi.org/project/eval-type-backport/
+    "eval-type-backport>=0.2.0",
```

Comment:
This basically fixes `pydantic` + `__future__ import annotations` for older python versions. I suspect this was why we have the exclusion in `pyproject.toml` to not upgrade to new style annotations in:
```
# Pydantic also require models to be imported during execution
"airflow/serialization/pydantic/*.py" = ["I002", "UP007", "TCH001"]
```

Reply:
Nice!